config/manager/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cockroach-operator
+  labels:
+    app: cockroach-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cockroach-operator
+  template:
+    metadata:
+      labels:
+        app: cockroach-operator
+    spec:
+      serviceAccountName: cockroach-operator-sa
+      containers:
+        - name: cockroach-operator
+          image: cockroachdb/cockroach-operator:latest
+          imagePullPolicy: IfNotPresent
```

Comment:
Side note, we may want to change this.

Reply:
I'm assuming you mean the image (and not the policy). The `:latest` tag here is just a placeholder. All of our codegen overrides this with the specific latest release. See _config/manager/patches/image.yaml_ which is generated from a template.